Included option to skip log file header

ckjblue opened the pull request:
The current code includes a header with the log file and prevents the output from being processed in an automated build process.  By default, the header will continue to be included in the log file.

> -skipheader [Exclude the header from the log file (default: false)]

rolfbjarne commented on Touch.Server/Main.cs:
```diff
@@ -99,20 +99,23 @@ public int Start ()
-			fs.Flush ();
+			if (!skipHeader)
+			{
+                // a few extra bits of data only available from this side
```
The coding style is to add the opening brace at the end of the `if` line:

```csharp
if (...) {
```

Also you're reformatting the subsequent lines (replacing tabs with spaces, removing spaces before the opening parenthesis in method calls, etc.). Please undo those changes.

spouliot commented on Touch.Server/Main.cs:
```diff
@@ -99,20 +99,22 @@ public int Start ()
+                byte[] array = Encoding.UTF8.GetBytes(header);
+                fs.Write (array, 0, array.Length);
+                fs.Flush ();
+            }
```
^ the alignment remains incorrect
you seem to be used spaces instead of tabs

code formatting is still incorrect, e.g. you removed the spaces before the `(`

mandel-macaque commented:
Love the idea. We need to also be able to skip it via env vars.

ckjblue commented:
Using the default settings for Visual Studio for Mac the setting was to convert tabs into spaces.  I updated the project file locally to leave the Tab as entered and corrected the alignment.  I hope this makes the grade.
